src/Impostor.Api.Innersloth.Generator/Impostor.Api.Innersloth.Generator.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <DebugType>embedded</DebugType>
+    <Nullable>enable</Nullable>
+    <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.7.0" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.4" PrivateAssets="all" />
+    <PackageReference Include="PolySharp" Version="1.13.2" PrivateAssets="all" />
```

Comment:
Do we need PolySharp or should we bump the TargetFramework to something newer?

Reply:
ugh, found my answer already:
>  Important: Currently only .NET Standard 2.0 assemblies can be used as Source Generators.
https://learn.microsoft.com/en-us/dotnet/csharp/roslyn-sdk/source-generators-overview

Reply:
Visual Studio still runs source generators in the main net framework process so unless we want to drop support for it, we have to stay with netstandard2.0